Libraries/Components/Button.windows.js

```diff
@@ -118,7 +118,22 @@ class Button extends React.Component {
      * @platform windows
      */
     onFocus: PropTypes.func,
-};
+    /**
+     * Determines control accessibility behavior.
+     * For more details refer to Libraries\Components\View\ViewPropTypes.js
+     * @platform windows
+     */
+    importantForAccessibility: PropTypes.oneOf([
+      'auto',
+      'yes',
+      'no',
+      'no-hide-descendants'
+    ]),
```

Comment:
There's no point on having this property! There's no configurable descendants!
Should't we just keep the default?

Reply:
@reseul The FocusableView has Text as a child. The default for the property is 'yes' but the user sometimes may want turn it off, so I guess we should keep it.

Reply:
@reseul After talking in person decided to just set it to the default and remove the exposed property.